animation_nodes/nodes/color/c_utils.pyx

```diff
@@ -0,0 +1,27 @@
+# cython: profile=True
+cimport cython
+
+from ... data_structures cimport (
+    ColorList,
+    VirtualColorList,
+    PolygonIndicesList
+)
+
+def loopsColorsList(long Amount, PolygonIndicesList polygons, VirtualColorList colors, 
+                    ColorList loopsColors):
```

Comment:
Don't pass the output list, just create it in Cython and return it.

Reply:
Now, it is in the Cython code.